src/index.tsx

```diff
@@ -61,7 +60,7 @@ function ReactP5WrapperComponent<Props extends SketchProps = SketchProps>({
   const wrapperRef = React.createRef<Wrapper>();
   const canvasInstanceRef = React.useRef<P5CanvasInstance<Props>>();
 
-  useIsomorphicEffect(() => {
+  React.useEffect(() => {
```

Comment:
How about making the use of `rooks` optional? 

By default we use `UseEffect`, and by passing the parameter you can use `useIsomorphicEffect`.

WDYT? Or bundle size more important from your point of view?

Reply:
Actually the server tests work as expected with the regular use effect at the moment and so an isomorphic effect isn't required. This is because of the changes we made to move to refs meaning everything can be done client side and all we render server side is the wrapper div element.

Reply:
You could, if you really wanted to, test with a nextjs app to see for yourself how it would work as part of your testing of the PR but it's up to you ☺️

Reply:
I don't want to go crazy like that, so I think we can merge it as it is for now.